src/index.html

```diff
@@ -4,19 +4,112 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, initial-scale=1.0"
     />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
+    <link
+      rel="stylesheet"
+      href="style.css"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
     />
-    <title>Moyo header</title>
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
       rel="stylesheet"
-      href="./style.css"
     />
+    <title>MOYO</title>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="head">
+      <a href="index.html">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="MOYO"
+        />
+      </a>
+
+      <nav>
+        <ul class="list">
+          <li class="link">
+            <a
+              href="index.html"
+              class="is-active item"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="link">
+            <a
+              href="index.html"
+              class="item"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="link">
+            <a
+              href="index.html"
+              class="item"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="link">
+            <a
+              href="index.html"
+              class="item"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="link">
+            <a
+              href="index.html"
+              class="item"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="link">
+            <a
+              href="index.html"
+              class="item"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="link">
+            <a
+              href="index.html"
+              class="item"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="link">
+            <a
+              href="index.html"
```

Comment:
All navigation links have the same 'href' attribute pointing to 'index.html'. This might not be intentional as it would lead all links to the same page. Consider updating each 'href' to point to the correct page or section.